### DISCUSSION TITLE
Copilot Education for Post-doctoral researchers?

### DISCUSSION THREAD

**Discussion by danieljrichard** [community/community#138650] [Copilot Conversations]
### Select Topic Area

Question

### Body

Hello,
I'm enrolled as a post-doctoral researcher at Stanford University, and wondered if I'd be able to apply for the Copilot Education program, given that I'm working at an academic institution in the US. Any information/advice would be greatly appreciated!

Thanks,

Daniel

**Comment by pg56714** [comment]
@danieljrichard 

https://education.github.com/discount_requests/application
First, you need to apply for GitHub Education, and then proceed with the application.

Mark me as answer thanks

[copilot](https://github.com/features/copilot)

Click **Get started with Copilot**.

Click **Start a free trial**.

That's awesome! It really is free to use, with the big text saying, **Congratulations! You are eligible to use GitHub Copilot for free**.

![image](https://github.com/user-attachments/assets/3db0309c-4888-435a-9378-e57d850093c2)
![image](https://github.com/user-attachments/assets/bb49cfe9-4cde-460e-9503-a38b28fb3648)
![image](https://github.com/user-attachments/assets/d3fa07f3-d37f-4561-9d40-39c0f879fdf7)

**Reply by artinmajdi** [reply]
This isn't as simple as you make it seem. Postdocs don't have any of the documents that GitHub requests. GitHub Education was available to me while I was a PhD student, but I haven't been able to access it as a postdoc.

**Reply by jyw-atgithub** [reply]
I am wondering about this, too.

**Reply by Roderick-SM** [reply]
same

**Reply by sercharpak** [reply]
Same. I guess at the moment it is not
